tools/src/scie_pants/install_pants.py

```diff
@@ -76,19 +75,20 @@ def install_pants_from_pex(
     venv_dir: Path,
     prompt: str,
     version: Version,
+    pex_url: str,
     ptex: Ptex,
     extra_requirements: Iterable[str],
-    find_links: str | None,
     bootstrap_urls_path: str | None,
 ) -> None:
     """Installs Pants into the venv using the platform-specific pre-built PEX."""
-    uname = os.uname()
-    pex_name = f"pants.{version}-cp39-{uname.sysname.lower()}_{uname.machine.lower()}.pex"
-
-    pex_url = f"https://github.com/pantsbuild/pants/releases/download/release_{version}/{pex_name}"
+    pex_name = pex_url.rsplit("/", 1)[-1]
     if bootstrap_urls_path:
```

Comment:
I think potentially this whole codepath should be removed? We should just trust `pex_url` to have been computed by the appropriate bootstrap-URLs look-ups earlier and don't need to re-do it. When removing this, we should keep the error messages, i.e. they'll need to move somewhere else.

Reply:
moved all bootstrap urls path checking to pants_version, invoked during configure pants..